test/EncumberableToken.t.sol

```diff
@@ -196,6 +196,35 @@ contract EncumberableTokenTest is Test {
         assertEq(wrappedToken.balanceOf(charlie), 40e18);
     }
 
+    function testTransferFromRevertsIfYouSpendEncumberedTokens() public {
```

Comment:
Just to double check, this test will fail for the previous implementation right?

Reply:
yep 😞